code generators/de.fhdo.lemma.model_processing.code_generation.container_base/pom.xml

```diff
@@ -0,0 +1,106 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Is this POM capable of creating a standalone runnable JAR of the generator? I fear not, because it lacks a specification for the Maven Assembly Plugin. In general: All code generators should be standalone runnable. And their build process should also yield a runnable JAR. You may take a look at this point as an entrypoint: https://stackoverflow.com/questions/574594/how-can-i-create-an-executable-jar-with-dependencies-using-maven

Requirements towards the standalone JAR:
- JAR classifier = `standalone`
- No deployment to Nexus repository (i.e., the Maven Assembly Plugin must only be invoked in the Package build phase but not in the deployment phase).